cmd/gpu_fakedev/gpu_fakedev.go

```diff
@@ -58,9 +59,11 @@ const (
 	devfsPath  = "dev"
 	mib        = 1024.0 * 1024.0
 	// null device major, minor on linux.
-	devNullMajor = 1
-	devNullMinor = 3
-	devNullType  = unix.S_IFCHR
+	devNullMajor    = 1
+	devNullMinor    = 3
+	devNullType     = unix.S_IFCHR
+	maxK8sLabelSize = 63
```

Comment:
This (+ rerunning `gofmt`) would reduce commit lines:
```suggestion
	// GPU connectivity
	maxK8sLabelSize = 63
```